### DISCUSSION TITLE
What do you think about defining an X86 target including WLAN?

### DISCUSSION THREAD

**Discussion by ChristianMiddendorf** [freifunk-gluon/gluon#3379] [Ideas]
The idea would be to define an X86 target with all common WLAN drivers and their firmware, as well as the USB-LAN drivers, in order to be able to easily create very powerful access points based on existing hardware.
In the future, low-cost WLAN7 APs with powerful network ports in particular could be provided in this way.
With X86-64 as the basis, a lack of memory should not be an argument.
This target would make it very easy to see whether your own hardware is supported.

**Comment by blocktrron** [comment]
Already the case for all x86 targets. If drivers for common MTK / ath radios are missing feel free to open a PR.

https://github.com/freifunk-gluon/gluon/blob/main/targets/x86.inc#L32

**Reply by ChristianMiddendorf** [reply]
Sorry, I did not checked it there.
So the docs at… x86 support[](https://gluon.readthedocs.io/en/latest/user/x86.html#x86-support)

Gluon can run on normal x86 systems, for example virtual machines and VPN boxes. By default, there is no WLAN support on x86 though.

have to be updated.
…and I have to find out why my sticks do not show up :)

Thank you!
